template.yaml

```diff
@@ -96,38 +103,81 @@ Parameters:
     Type: String
     Default: IdentityReuseService
 
+  MainStackName:
+    Description: The name of the Main VC Storage stack
+    Type: String
+    Default: identity-reuse-service-main
+
 Mappings:
   EnvironmentMap:
     "local":
       txmaAccountId: "275907361037"
       dynatraceSecretArn: "arn:aws:secretsmanager:eu-west-2:216552277552:secret:DynatraceNonProductionVariables"
       messageQueueArn: "unused"
       messageQueueKeyArn: "unused"
+      ApiRateLimit: 5         # allowed requests per second - if updating this value review lambda concurrency values below
+      ApiBurstLimit: 5        # requests the API can handle concurrently - if updating this value review lambda concurrency values below
+      IPVCoreVPCID: vpc-093ac0375aa3f291d # ID Reuse Core Dev VPC ID
+      OneLoginHomeVPCID: vpc-093ac0375aa3f291d # ID Reuse Core Dev VPC ID
```

Comment:
Another copy/paste issue ... can we lose all the reference to `OneLoginHomeVPCID` thats very EVCS specific is never going to be needed in the SIS world.